index.js

```diff
@@ -96,7 +98,27 @@ module.exports = async params => {
 	}
 
 	const {packageJson} = readPkgUp.sync();
-	const {appId} = packageJson.build;
+	const buildConfig = packageJson.build;
+	let appId = '';
+
+	if (buildConfig) {
+		// appId from ./package.json
+		appId = buildConfig.appId;
+	} else {
+		try {
+			// fallback for vue-cli-plugin-electron-builder
+			// it uses vue.config.js to create electron builder config yaml file
+			// https://github.com/nklayman/vue-cli-plugin-electron-builder
+			const path = './dist_electron/builder-effective-config.yaml'
+			if (fs.existsSync(path)) {
+				let fileContents = fs.readFileSync(path, 'utf8');
+				let data = yaml.safeLoad(fileContents);
+				appId = data.appId;
+			}
+		} catch(err) {
```

Comment:
If you agree to leave this way as the only one of finding the appId, then we should throw instead of console logging.